code/modules/mob/living/basic/lavaland/watcher/watcher.dm

```diff
@@ -0,0 +1,116 @@
+/// A floating eyeball which keeps its distance and plays red light/green light with you.
+/mob/living/basic/mining/watcher
+	name = "watcher"
+	desc = "A levitating, monocular creature held aloft by wing-like veins. A sharp spine of crystal protrudes from its body."
+	icon = 'icons/mob/simple/lavaland/lavaland_monsters_wide.dmi'
+	icon_state = "watcher"
+	icon_living = "watcher"
+	icon_dead = "watcher_dead"
+	health_doll_icon = "watcher"
+	pixel_x = -12
+	base_pixel_x = -12
+	speak_emote = list("chimes")
+	speed = 3
+	maxHealth = 200
+	health = 200
+	attack_verb_continuous = "buffets"
+	attack_verb_simple = "buffet"
+	crusher_loot = /obj/item/crusher_trophy/watcher_wing
+	ai_controller = /datum/ai_controller/basic_controller/watcher
+	faction = list(FACTION_WATCHER)
```

Comment:
i think this will get overridden by the spawner component on the tendil-spawned watchers and they will fight mapgen watchers. sounds funny though